public/js/app/views/BookmarkletView.js

```diff
@@ -18,6 +18,14 @@ define(["app/app",
       })
     },
 
+    selectInstagramThumbnail: function() {
+      var re = /https?:\/\/www\.instagram\.com\/p\/([\w-]{11})\//i;
```

Comment:
11 is a wrong number, Instagram ID's doesn't have a fixed length. For example, first image in Instagram: https://www.instagram.com/p/C/. So I suggest to use '+', not '{11}'.